keras/backend/theano_backend.py

```diff
@@ -111,6 +111,13 @@ def dot(x, y):
     return T.dot(x, y)
 
 
+def batch_dot(x, y, axes=None):
+    if axes is None:
+        # behaves like tf.batch_matmul as default
```

Comment:
Yes, as long as the different backends behave in the same way, it does not matter whether the original behavior is from TH or TF (or Numpy...).